config/locales/en.yml

```diff
@@ -237,6 +237,13 @@ en:
         save_user_status_error: "Saving failed. Please refresh the page and try again."
         ban_user_confirmation: "This removes the user from the marketplace and prevents them from accessing the site again with this account. Are you sure you want to proceed?"
         ban_me_error: "You cannot ban yourself."
+      new_layout:
+        new_layout: "New layout"
+        description_roadmap: "You can choose whether you want to use the new layout or the old layout for your marketplace. It's recommended to use the new layout, since most of our new features will not support the old layout. The old layout will be eventually removed entirely (you'll be notified when that happens)."
+        description_process: "The new layout is released one page at a time. You can decide to keep the old layout for some pages and use the new layout for others. You can preview all the pages without showing them to your users by only enabling the new pages for yourself."
```

Comment:
I'm not sure if "one page at a time" is actually the best description: Our first element, new top bar, already contradicts it. Maybe one view or element at a time?